script/vsts/platforms/templates/cache.yml

```diff
@@ -11,20 +11,20 @@ steps:
   - task: Cache@2
     displayName: Cache node_modules
     inputs:
-      key: 'npm | "$(Agent.OS)" | "$(BUILD_ARCH)" | package.json, package-lock.json, script/vsts/platforms/${{ parameters.OS }}.yml'
+      key: 'npm_pr_78 | "$(Agent.OS)" | "$(BUILD_ARCH)" | package.json, package-lock.json, script/vsts/platforms/${{ parameters.OS }}.yml'
```

Comment:
What is this?

Reply:
That should be removed before this would be merged.

That's setting the cache aside for this PR so it doesn't pull anything in from `master` branch, or even earlier commits of the PR. Nothing else has that cache `key` value, so there is no existing cache to hit.

I was seeing weird errors, so rather than spend time debugging them, I wanted to see if they went away when cache restoring was disabled.

Next step is to remove that and see if it passes CI when restoring cache from the general pool (e.g. from `master`) is enabled again.

Reply:
Please revert this. If there is a problem with caching we need to fix that instead,